cmd/podman/run.go

```diff
@@ -203,7 +203,7 @@ func runCmd(c *cli.Context) error {
 		return err
 	}
 
-	if ecode, err := ctr.ExitCode(); err != nil {
+	if ecode, _, err := ctr.ExitCode(); err != nil {
```

Comment:
I would want this checked and the exit code set to 125 if the command is completing without the container exit code.

Reply:
Or at lease the ecode ignored.

Reply:
Run/start shouldn't need this once #1086 gets in and they switch to Wait - we have a guarantee of a good exit code